src/utility/firebase.js

```diff
@@ -8,13 +8,13 @@ import { getAnalytics } from 'firebase/analytics';
 // Your web app's Firebase configuration
 // For Firebase JS SDK v7.20.0 and later, measurementId is optional
 const firebaseConfig = {
-  apiKey: 'AIzaSyDs-rxvOLmskYhudQWPOxl-whELlKKUYhs',
-  authDomain: 'issue-finder.firebaseapp.com',
-  projectId: 'issue-finder',
-  storageBucket: 'issue-finder.appspot.com',
-  messagingSenderId: '918268898722',
-  appId: '1:918268898722:web:653b04454a9eee41d896fb',
-  measurementId: 'G-C28740E7D5'
+  apiKey:process.env.REACT_APP_API_KEY,
```

Comment:
write this line **apiKey:process.env.REACT_APP_API_KEY** as  **apiKey:process.env.REACT_APP_FIRBASE_API_KEY**